tests/configs/test_configs.py

```diff
@@ -4,8 +4,76 @@
 import google.protobuf.text_format as text_format  # weird import for mypy
 import pytest
 from myrtlespeech import configs
+from myrtlespeech.builders.task_config import build
 from myrtlespeech.protos import task_config_pb2
 
+# Utilities -------------------------------------------------------------------
+
+DATASET_STR = "dataset "
+FAKE_DSET = """{
+fake_speech_to_text {
+    dataset_len: 2;
+    audio_ms {
+        lower: 1;
+        upper: 10;
+        }
+    label_symbols: "abc";
+    label_len {
+        lower: 1;
+        upper: 10;
+        }
+    }
+}"""
+
+
+def replace_dataset_w_fake_dataset(config):
```

Comment:
I was also going to ask you the same thing. Maybe also some simple regex could solve this problem, but i think Sam's solution would be quicker to implement.